### PR TITLE
Upper and Lower bounds

### PR DESCRIPTION
**Pull request by do-ruth**
@quietHamster
The basics of my code work fine but now I want to start checking for invalid inputs. I edited this code so that if the user inputs an upper bound less than the lower bound, the program will repeatedly ask the user for a valid upper bound until one is givenl.

### CODE REVIEW

File: src/RandNumGenerator.java [hunk 1]
```diff
@@ -17,20 +17,33 @@ public static void main (String[] args) {
...
-        while (i > 0) {
-            int randNum = lowerNum + rand.nextInt(upperNum - lowerNum);
-            System.out.println(randNum);
-            i--;
+        int invalNum = 1;
+        int upperNum;
+        while (invalNum != 0) {
+            System.out.println("Please input the upper bound of your range (i.e. 10)");
+            upperNum = mrScanner.nextInt();
+            if (upperNum < lowerNum) {
```
Comment (quietHamster) [inline src/RandNumGenerator.java]: What happen when `upperNum equals lowerNum`

File: src/RandNumGenerator.java [hunk 2]
```diff
@@ -17,20 +17,33 @@ public static void main (String[] args) {
...
+                System.out.print("Your upper bound is less than your lower bound. Please input a valid integer.");
+            }
+            else {
+                invalNum = 0;
+                System.out.println("The upper bound is: " + upperNum);
+                //getting amount of numbers
+                System.out.println("How many random numbers would you like?");
+                int amtNum = mrScanner.nextInt();
+                System.out.println("Generating " + amtNum + " number(s):");
```
Comment (quietHamster) [inline src/RandNumGenerator.java]: What happen if user input `-1`

**Comment by quietHamster** [comment]
Other design questions:
Use case: `upper = 10, lower = 9`, what will be the result?
Use case: `upper = 10, lower = 8`, what will be the result?

Please note: YOU HAVE A GREAT INITIAL CODE!
I am pushing harder to the boundary, so walking together!

File: src/RandNumGenerator.java (hunk 1 above)
Comment (do-ruth) [reply src/RandNumGenerator.java]: I didn't even think about that!

**Pull request closed without merging**